spp_change_request/models/mixins/source_mixin.py

```diff
@@ -584,7 +583,7 @@ def _copy_service_point_ids(self, change_request_field):
                     change_request_field: rec.id,
                     "service_point_id": mrec.id,
                 }
-                self.env["pds.change.request.service.point"].create(service_points)
+                self.env["spp.change.request.service.point"].create(service_points)
```

Comment:
@gonzalesedwin1123 upon checking, there is no existing model with the name `spp.change.request.service.point` in OpenSPP

Reply:
Yes but I plan to keep this function as I need to scan all that might still be using this. I will have this function removed once I am done checking everything that might be referring to this function.

Reply:
ah, i see, I think it is better to comment it as well, then add a TODO note, something like below code

```
# TODO: Investigate this
# self.env["spp.change.request.service.point"].create(service_points)
```